tutorials/epochs/plot_10_epochs_overview.py

```diff
@@ -1,103 +1,390 @@
+# -*- coding: utf-8 -*-
 """
 .. _tut-epochs-class:
 
-The :class:`Epochs <mne.Epochs>` data structure: epoched data
-=============================================================
+The Epochs data structure: discontinuous data
+=============================================
 
-:class:`Epochs <mne.Epochs>` objects are a way of representing continuous
-data as a collection of time-locked trials, stored in an array of shape
-``(n_events, n_channels, n_times)``. They are useful for many statistical
-methods in neuroscience, and make it easy to quickly overview what occurs
-during a trial.
+This tutorial covers the basics of creating and working with :term:`epoched
+<epochs>` data. It introduces the :class:`~mne.Epochs` data structure in
+detail, including how to load, query, subselect, export, and plot data from a
+:class:`~mne.Epochs` object. For more info on visualization of
+:class:`~mne.Epochs` objects, see :ref:`tut-visualize-epochs`. For info on
+creating an :class:`~mne.Epochs` object from (possibly simulated) data in a
+:class:`NumPy array <numpy.ndarray>`, see :ref:`tut_creating_data_structures`.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+As usual we'll start by importing the modules we need:
 """
 
+import os
 import mne
-import os.path as op
-import numpy as np
-from matplotlib import pyplot as plt
 
 ###############################################################################
-# :class:`Epochs <mne.Epochs>` objects can be created in three ways:
-#  1. From a :class:`Raw <mne.io.Raw>` object, along with event times
-#  2. From an :class:`Epochs <mne.Epochs>` object that has been saved as a
-#     `.fif` file
-#  3. From scratch using :class:`EpochsArray <mne.EpochsArray>`. See
-#     :ref:`tut_creating_data_structures`
+# :class:`~mne.Epochs` objects are a data structure for representing and
+# analyzing equal-duration chunks of the EEG/MEG signal. :class:`~mne.Epochs`
+# are most often used to represent data that is time-locked to repeated
+# experimental events (such as stimulus onsets or subject button presses), but
+# can also be used for storing sequential or overlapping frames of a continuous
+# signal (e.g., for analysis of resting-state activity). Inside an
+# :class:`~mne.Epochs` object, the data are stored in an :class:`array
+# <numpy.ndarray>` of shape ``(n_epochs, n_channels, n_times)``.
+#
+# :class:`~mne.Epochs` objects have many similarities with :class:`~mne.io.Raw`
+# objects, including:
+#
+# - They can be loaded from and saved to disk in ``.fif`` format, and their
+#   data can be exported to a :class:`NumPy array <numpy.ndarray>` through the
+#   :meth:`~mne.Epochs.get_data` method or to a :class:`Pandas DataFrame
+#   <pandas.DataFrame>` through the :meth:`~mne.Epochs.to_data_frame` method.
+#
+# - :class:`~mne.Epochs` objects support channel selection by index or name,
```

Comment:
On version of .circle-artifact that I generated (but for some reason not here or a later commit), many of these lines start with "Both Epochs and Raw objects...".  That seems redundant to me, as the paragraph lead already states that this is a list of similarities between Epochs and Raw objects.

Reply:
I've addressed all your comments (I think) except for this one.  I agree that it's a bit redundant, but an earlier version of this PR didn't have the redundancy and I thought it was better to add it in.  I thought it was good reinforcement.